README.md

```diff
@@ -155,7 +176,7 @@ connection.execute(query: query, parameters: supplied_key1, supplied_key2) { que
 
 If your application executes some query multiple times with different parameters, you may want to improve the performance of the application by using a prepared statement for that query. I.e. send the query to the database in advance, and later use the returned handle to execute the query providing the parameters. This way the database server will process the query only once.
```

Comment:
Some new text for this section -

If your application executes the same (or similar) SQL statements repeatedly with different parameters you may improve the performance of the application by using a prepared statement. Prepared statements can reduce parsing time as the database parses and compiles the statement template only once and then stores the result but doesn’t execute it. Later, the application supplies values for the parameters of the statement template and the database executes the statement.

For example, suppose our application needs to retrieve the average grade for courses with an average above a given value; a value which we want to vary.  Let’s change our query to use a parameter instead of a predefined value:

Line 188 - please change to "Now, prepare the statement:"

Line 201 - please change to "Now the application may execute the prepared statement as many times as it wants with different parameter values:"